src/sagas/index.ts

```diff
@@ -7,8 +7,8 @@ import * as actions from '../actions'
 import * as actionTypes from '../actions/actionTypes'
 import { IState } from '../reducers/states'
 import { Context, interrupt, runInContext } from '../slang'
-import { history } from '../utils/history'
 import { showSuccessMessage, showWarningMessage } from '../utils/notification'
+import { IVLE_API_KEY } from '../utils/secrets'
```

Comment:
1. We should request an API key specifically for this frontend rather than use our own student ones (obviously).
2. I considered hashing this, but it'll be exposed through the browser GET call anyways.